impl/implementation.go

```diff
@@ -27,111 +28,106 @@ const (
 
 type guidMap map[string]uuid.UUID
 type cdMap map[string]implementation.ComponentDefinition
-
 type component struct {
-	id             string
-	compNameIndex  int
-	name           string
-	uuidIndex      int
-	narrativeIndex int
-	definition     cdMap
+	id                   string
+	compNameIndex        int
+	name                 string
+	parameterIDIndex     int
+	parameterStringIndex int
+	uuidIndex            int
+	narrativeIndex       int
+	definition           cdMap
+	hasParameterMapping  bool
 }
 
-// GenerateImplementation generates implementation from component excel sheet
-func GenerateImplementation(CSVS [][]string, p *profile.Profile, c Catalog) implementation.Implementation {
+var profileMap = map[string]string{
```

Comment:
@minhaj10p is there a reason why we're duplicating these profile entries?

Reply:
the excel sheet had different variations of the same name. this map sanitizes it for the fedramp profiles

Reply:
Gotcha. Thanks.